WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -438,15 +445,10 @@ + (NSString *)deviceNameByUserInterfaceIdiom:(UIUserInterfaceIdiom) userInterfac
 
 + (NSDictionary *)currentCapabilities
 {
-  FBApplication *application = [FBSession activeSession].activeApplication;
```

Comment:
I don't think this is very helpful because this API is also invoked in multiple other places and could slow them down. It is necessary to figure out which exact API performs a slow call and deal with that

Reply:
Yes, this pr is only for a new session request case, actually not the root cause fix.

The root is probably https://github.com/appium/WebDriverAgent/blob/5d2bd5fe4228cfa624f8700619879b62012ceca8/WebDriverAgentLib/FBApplication.m#L118-L123 , or `.activeApplication` after getting the response.

I'll take a look further around there (since currently I have a device that has this issue)

Reply:
then, will create another issue or some findings separately

Reply:
not with breakpoint observation, but this slowness occurred with `com.apple.springboard`. I haven't seen similar things except for that right now, but other apps also could happen of course i assume.

Anyway, this is not necessary as `handleCreateSession` usage in current xcuitest driver, so i made this pr for new session request usage scope.

Reply:
Btw, do you have exact steps to reproduce this issue? It for sure does not block on simulators in my tests

Reply:
oo, my account reached the maximum device count.... I'll check further as another investigation work when my account back to usable again

Reply:
No, unfortunately... It is kind of a random thing. For example, after a device reboot, I no longer see this behavior. But after a while? I heard this behavior occurred again. Even without a device reboot, this behavior could be gone. But once started, this behavior continued for a while.

So far, I only observed this behavior in real devices.

Reply:
In the device log, after `Getting the most recent active application` message, no WDA related system log appeared for a while so I tried to add log like https://github.com/appium/WebDriverAgent/pull/772 to check the possible cause.

Then today, I saw this behavior with `http://127.0.0.1:8100/wda/activeAppInfo` endpoint call as well, so future investigation will be via the endpoint

Reply:
I'll leave new findings in https://github.com/appium/appium/issues/19186 by checking the activeAppInfo endpoint

Reply:
maybe try https://github.com/appium/WebDriverAgent/pull/776 ?

Reply:
let me try

Reply:
no change, unfortunately. After `Getting the most recent active application` message, i opened preference app, then the response (sessionInformation) responded. so maybe something ios internal is related i guess